clear-containers-image/update_image.sh

```diff
@@ -10,6 +10,10 @@ set -e
 source ../versions.txt
 source ../scripts/pkglib.sh
 
+# get versions from clearcontainers/runtime repository
+runtime_versions_url="https://raw.githubusercontent.com/clearcontainers/runtime/master/versions.txt"
```

Comment:
intead of master you may want to point to the version you want to release?

Reply:
Oops - sorry @jcvenegas! That's a good point though.